keyboards/minidox/keymaps/bepo/keymap.c

```diff
@@ -0,0 +1,111 @@
+#include QMK_KEYBOARD_H
+#include "keymap_bepo.h"
+
+extern keymap_config_t keymap_config;
+
+#define TAPPING_TERM 450
```

Comment:
This should go within a `config.h`

Reply:
> 
> 
> This should go within a `config.h` for it to take effect, however i cannot make an automatic suggestion to add it.

Added config.h, but I removed this config.